homeassistant/components/climate/ephember.py

```diff
@@ -57,13 +57,22 @@ def __init__(self, ember, zone):
         self._zone = zone
         self._hot_water = zone['isHotWater']
 
+    @property
+    def state(self):
```

Comment:
Don't make this the `state` method. The previous implementation was correct - this should be returned from the `current_operation` method.

Reply:
In that case the state will always be "auto". I would prefer to have the state be if the boiler is active. I could make that a state attribute instead of the main state but would prefer it to be the main state.

Reply:
You are never allowed to override the `@state` property unless you are implementing a sensor entity.